src/code/z_camera.c

```diff
@@ -7843,18 +7848,25 @@ void Camera_Finish(Camera* camera) {
 
 s32 func_8005A02C(Camera* camera) {
     camera->stateFlags |= (CAM_STATE_2 | CAM_STATE_3);
-    camera->stateFlags &= ~(CAM_STATE_3 | CAM_STATE_12);
+    camera->stateFlags &= ~(CAM_STATE_3 | CAM_STATE_DEMO7);
     return true;
 }
 
+#define CAM_CHANGE_MODE_0 (1 << 0)
+#define CAM_CHANGE_MODE_1 (1 << 1)
+#define CAM_CHANGE_MODE_2 (1 << 2)
+#define CAM_CHANGE_MODE_3 (1 << 3)
+#define CAM_CHANGE_MODE_4 (1 << 4)
+#define CAM_CHANGE_MODE_5 (1 << 5)
+
 s32 Camera_ChangeModeFlags(Camera* camera, s16 mode, u8 flags) {
-    static s32 modeChangeFlags = 0;
+    static s32 sModeChangeFlags = 0;
```

Comment:
iirc, the only reason I waited to not name these was because we still need to apply a cam mode name change discussed in discord a while back, something like `CAM_MODE_TARGET` -> `CAM_MODE_ZPARALLEL` (or alternatives, can't remember what we settled on) since `TARGET` is not z-targeting as one would expect, instead `CAM_MODE_FOLLOW_TARGET` is. I'll be doing that soon:tm:

And as a reminder, we also had something decided for `FIXED`/`PREREND`/`PINNED` cam settings. or something, need to pull out old convos. Can't remember exactly what we came up with

Reply:
Good catch with the sfxId.

Also yeah, `sModeChangeFlags = 0;` is the first usage of this variable, so any previous values gets reset. So being static doesn't do anything

Reply:
Thanks for reminding me about those names being wip, I thought they were final

Alright, later then for the naming

Reply:
I take it we now wait on #1455 before getting back to this discussion/PR ?

Reply:
it's a bit hidden I guess woops
In a comment above ( https://github.com/zeldaret/oot/pull/1453#discussion_r1040166361 ) I suggest changing "`Camera_ChangeModeFlags`'s `flags` to something like `forceModeChange`"

Reply:
With #1455 merged we can get back to this!

Reply:
I'm still not seeing your original point, what's the connection between `flags`/`forceModeChange` (i.e. arg2) and `sModeChangeFlags`. If `flags` is changed to `forceModeChange`, that doesn't impact `sModeChangeFlags` at all? What's your suggested rename of `sModeChangeFlags`?